src/voice/handler.rs

```diff
@@ -212,6 +212,7 @@ impl Handler {
         // Only send an update if we were in a voice channel.
         if self.channel_id.is_some() {
             self.channel_id = None;
+            self.send(VoiceState::Disconnect);
```

Comment:
Right idea, because this will stop the voice thread from trying to send over a broken socket. But this doesn't actually compile.

Reply:
```suggestion
            self.send(VoiceStatus::Disconnect);
```